src/api/services/emails/emailProvider.js

```diff
@@ -0,0 +1,80 @@
+const nodemailer = require('nodemailer');
+const { emailConfig } = require('../../../config/vars');
+const Email = require('email-templates');
+
+
+// SMTP is the main transport in Nodemailer for delivering messages.
+// SMTP is also the protocol used between almost all email hosts, so its truly universal.
+// if you dont want to use SMTP you can create your own transport here
+// such as an email service API or nodemailer-sendgrid-transport
+
+const transporter = nodemailer.createTransport({
+  port: emailConfig.port,
+  host: emailConfig.host,
+  auth: {
+    user: emailConfig.username,
+    pass: emailConfig.password,
+  },
+  secure: false, // upgrades later with STARTTLS -- change this based on the PORT
+});
+
+// verify connection configuration
+transporter.verify((error, success) => {
+  if (error) {
+    console.log('error with email connection', error);
+  } else {
+    console.log('Server is ready to take our emails', success);
+  }
+});
+
+exports.sendPasswordReset = async (passwordResetObject) => {
+  const email = new Email({
+    views: { root: __dirname },
+    message: {
+      from: 'support@your-app.com',
+    },
+    // uncomment below to send emails in development/test env:
+    send: true,
+    transport: transporter,
+  });
+
+  email
+    .send({
+      template: 'passwordReset',
+      message: {
+        to: passwordResetObject.email,
```

Comment:
```suggestion
        to: passwordResetObject.userEmail,
```

Reply:
@thisisdice thanks for catching that!

Reply:
@travislang thank you for the PR. Very useful addition!!